_data/projects/PySwarms.yml

```diff
@@ -0,0 +1,11 @@
+name: PySwarms
+desc: A Particle Swarm Optimization (PSO) library in Python
+site: https://github.com/ljvmiranda921/pyswarms
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- optimization
+- algorithms
+upforgrabs:
+  name: https://github.com/ljvmiranda921/pyswarms/labels/help%20wanted
```

Comment:
Oops, let's try this again. These two items are used to link off to a list of issues.

 - `name` should just be the label `help wanted` (no encoding)
 - `url` should be the link to the list of issues, not to a specific issue

We favour the list of issues because it's more granular and let's work on different areas happen concurrently. Hopefully that makes sense!